templates/services/backend/manifest.yml

```diff
@@ -9,8 +9,12 @@ name: {{.Name}}
 type: {{.Type}}
 
 image:
-  # Path to your service's Dockerfile.
-  build: {{.Image.Build}}
+  # Docker build arguments. You can specify Docker Compose-style overrides here. Supported: dockerfile, context, args
+  build:
+    # Path to your service's Dockerfile.
+    dockerfile: {{.Image.Build.BuildArgs.Dockerfile}}
+    # Path to the build context directory relative to workspace root.
+    context: {{ dirName .Image.Build.BuildArgs.Dockerfile}}
```

Comment:
I wonder if we should keep the old version by default:
```yaml
# Path to your service's Dockerfile. 
# This field can be also an object to specify build args and a context.
build: {{.Image.Build.BuildArgs.BuildString}}
```
This way we won't have to pass a `dirName` fn

Reply:
Yea - let's keep the old default!